tools/scylla-sstable.cc

```diff
@@ -2623,10 +2623,7 @@ void write_operation(schema_ptr schema, reader_permit permit, const std::vector<
     }
     auto input_file = vm["input-file"].as<std::string>();
     auto output_dir = vm["output-dir"].as<std::string>();
-    if (!vm.count("generation")) {
-        throw std::invalid_argument("missing required option '--generation'");
-    }
-    auto generation = sstables::generation_type(vm["generation"].as<int64_t>());
+    auto generation = sstables::generation_type(utils::UUID_gen::get_time_UUID());
```

Comment:
q: any benefit in using UUIDv7 here? (CC @bhalevy ) - would that be useful in any way?

Reply:
It's great for breaking compatibility